conference/react-js/src/components/AddNewOrganizer/AddNewOrganizer.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import Dialog from 'material-ui/Dialog';
+import RaisedButton from 'material-ui/RaisedButton';
+import MenuItem from 'material-ui/MenuItem';
+import SelectField from 'material-ui/SelectField';
+import TextField from 'material-ui/TextField';
+import { conferencesNames } from '../../constants/backend-url';
+
+class AddNewOrganizer extends Component {
+  constructor(prop) {
+    super(prop);
+    this.state = {
+      open: false,
+      value: false,
+      conferenceList: [],
+    };
+  }
+
+  componentDidMount() {
+    axios.get(conferencesNames)
+      .then(({ data }) => {
+        this.setState({ conferenceList: data });
+      });
+  }
+
+  handleOpen = () => {
```

Comment:
i think, You can write this two function in one. like this "this.setState({open: !this.state.open})"